crx_template/content.js

```diff
@@ -66,22 +71,34 @@
             delete ports[portName];
         });
     }
+    
+    
+    function initPage(evt) { //initializing firebreath object - and passing it to callback
+    	injectScript("window['" + evt.callback + "']((function() {" +
+        	"var firebreath = {extId: '" + extId + "'};" +
+        	includeScript("FireBreathPromise.js") +
+        	includeScript("firewyrm.js") +
+        	includeScript("wyrmhole.js") +
+        	includeScript("wyrmhole-page.js") +
+        	"return firebreath;" +
+        "})())");
+    }
+    
+    function includeScript(path) { //getting javascript as text
+        var request = new XMLHttpRequest();
```

Comment:
GET call does give a warning in Chrome console window.

Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.